docs/getting_started/install.md

```diff
@@ -17,6 +17,21 @@ make
 ```
 For example:  `-DLP_SOLVE=/usr/lib/lpsolve/liblpsolve55.so`
 
+In WSL (Windows Subsystem Linux), you can run the following command to install libc6-dev-i386. This will be required for `ieeefp.h` which is used under `qd` library,
+
+    sudo apt-get install libc6-dev-i386
+
+Also, to get the `mkl` support, run the following,
```

Comment:
"Also to install `mkl` related dependencies, run the following"

Reply:
Added in latest!